dist/tools/isrvecgen/isrvecgen.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
```

Comment:
why making this only python 3 ?

Reply:
no idea, thought I'll try to be 'modern' :-)

Reply:
Are there still major distros or OSs that do not supply python 3 (it has not to be the default)? Even the grand old ladies Debian and CentOS have python 3 now.

Reply:
I don't think mac OS users have python3 installed by default

Reply:
As this is not needed for building, having python3 as dependency should be fine.